saleor/graphql/account/types.py

```diff
@@ -310,6 +310,13 @@ class User(CountableDjangoObjectType):
         ),
         model_field="groups",
     )
+    editable_groups = gql_optimizer.field(
```

Comment:
Optimizer here is not needed because we don't use `user.groups` in the resolver anyway.